jmix-flowui/flowui/src/main/resources/io/jmix/flowui/view/layout.xsd

```diff
@@ -4221,6 +4221,17 @@
         </xs:complexContent>
     </xs:complexType>
 
+    <xs:complexType name="rangeInputComponent">
+        <xs:complexContent>
+            <xs:extension base="baseHtmlComponent">
+                <xs:attribute name="min" type="xs:double"/>
+                <xs:attribute name="max" type="xs:double"/>
+                <xs:attribute name="step" type="xs:double"/>
+                <xs:attribute name="orientation" type="xs:string"/>
```

Comment:
1. Not all direct attributes are added
2. `orientation` is enum, `string` isn't suitable type for it